addons/CLib/Events/fn_clientInit.sqf

```diff
@@ -115,22 +67,29 @@ GVAR(groupUnits) = [];
             [_name + "Changed", [_currentValue, _oldValue]] call CFUNC(localEvent);
             GVAR(EventNamespace) setVariable [_name, _currentValue];
         };
-    }, 0, [_x, _code]] call CFUNC(addPerFrameHandler);
+    }, 0, [_name, _code]] call CFUNC(addPerFrameHandler);
 
     true
 } count [
-    "currentThrowable",
-    "currentWeapon",
-    "vehicle",
-    "assignedVehicleRole", // This has to be after "vehicle"
-    "side",
-    "group",
-    "leader",
-    "getConnectedUAV",
-    "currentVisionMode"
+    ["player", {missionNamespace getVariable ["bis_fnc_moduleRemoteControl_unit", player]}],
+    ["currentThrowable", {currentThrowable CLib_Player}],
+    ["currentWeapon", {currentWeapon CLib_Player}],
+    ["vehicle", {vehicle CLib_Player}],
+    ["assignedVehicleRole", {assignedVehicleRole CLib_Player}], // This has to be after "vehicle"
+    ["side", {side CLib_Player}],
+    ["group", {group CLib_Player}],
+    ["leader", {leader CLib_Player}],
+    ["getConnectedUAV", {getConnectedUAV CLib_Player}],
+    ["currentVisionMode", {currentVisionMode CLib_Player}],
+    ["playerInventory", {CLib_Player call CFUNC(getAllGear)}],
+    ["visibleMap", {visibleMap}],
+    ["playerSide", {playerSide}],
+    ["cursorTarget", {cursorTarget}],
+    ["cursorObject", {cursorObject}],
+    ["units", {units CLib_Player}]
```

Comment:
this will break things in AAW and other mods i wrote please keep the backward compatible. this should be named grounUnitsChanged and not Units Changed. and i still think that the current name UnitsChanged is still not clear because it could also refer to allUnits